packages/tradershub/src/features/cfd/screens/DynamicLeverage/DynamicLeverageTableColumnHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { CaptionText, Text } from '@deriv/quill-design';
+
+type TDynamicLeverageTableColumnHeader = {
+    subtitle: string;
+    title: string;
+};
+
+export const DynamicLeverageTableColumnHeader: FC<TDynamicLeverageTableColumnHeader> = ({ subtitle, title }) => (
```

Comment:
Same file is in 
https://github.com/binary-com/deriv-app/pull/12748/files

Reply:
I created that PR's branch from this one that's why this file is showing it there. Once this PR is merged, this file won't show up on that PR.